sand/models/transformation.py

```diff
@@ -0,0 +1,45 @@
+from __future__ import annotations
+from typing import List, Union, Literal
+from peewee import (
+    CharField,
+    ForeignKeyField,
+    TextField,
+    IntegerField,
+    BooleanField,
+)
+from playhouse.sqlite_ext import JSONField
+
+from sand.models.base import BaseModel
+from sand.models.table import Table
+
+
+class Transformation(BaseModel):
+    name = CharField()
+    table = ForeignKeyField(Table, backref="transformations", on_delete="CASCADE")
+    mode = CharField()
+    datapath: Union[List[str], str] = JSONField()  # type: ignore
+    outputpath: List[str] = JSONField()  # type: ignore
```

Comment:
when using future, use list instead of List, | instead of Union.

Reply:
reverted it back to the older version as it is not compatible with python 3.8